### PR TITLE
Task solved

### PR DESCRIPTION
**Pull request by Sydorovich**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,47 @@
...
+        temp = temp * (self.clean_mark - car.clean_mark)
+        return round(temp / self.distance, 1)
+
+    def wash_single_car(self, car: Any) -> None:
+        if self.clean_mark <= car.clean_mark:
+            return
+        car.clean_mark = self.clean_mark
+
+    def rate_service(self, rate: int) -> None:
+        self.average_rating *= self.count_of_ratings
```
Comment (YuriiSmolii) [inline app/main.py]: Explain why do you need this multiplication.

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,47 @@
...
+        self.distance = distance_from_city_center
+        self.clean_mark = clean_mark
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Any) -> float:
+        if self.clean_mark <= car.clean_mark:
+            return 0.0
+        temp = car.comfort_class * self.average_rating
+        temp = temp * (self.clean_mark - car.clean_mark)
```
Comment (YuriiSmolii) [inline app/main.py]: This is bad code construction, simplify it.

**Review by YuriiSmolii — CHANGES_REQUESTED** [review]
Minor change and explanation requested.

File: app/main.py (hunk 1 above)
Comment (Sydorovich) [reply app/main.py]: Simplified it in new commit

File: app/main.py (hunk 2 above)
Comment (Sydorovich) [reply app/main.py]: Simplified it in new commit

**Comment by Sydorovich** [comment]
Pls recheck the task after the changes.

**Pull request closed without merging**